scripts/build.sh

```diff
@@ -2,25 +2,52 @@
 
 set -e
 
-PATH=$(npm bin):"$PATH"
-export PATH
 CLEANSLATE="node_modules/cleanslate/docs/files/cleanslate.css"
 
+isWindowsMingw() {
+  local is_mingw="False"
+  if [ "$(uname | cut -c 1-5)" == "MINGW" ]; then
+    is_mingw="True"
+  fi
+
+  echo -n "${is_mingw}"
+}
+
+if [ "$(isWindowsMingw)" == "True" ]; then
+  NPM_BIN_DIR="$(cygpath $(npm bin))"
+  PATH=$NPM_BIN_DIR:$PATH
+else
+  PATH="$(npm bin):$PATH"
+fi
+
+export PATH
+
+mkdir -p build
+mkdir -p build/static
 mkdir -p generated/static
 mkdir -p generated/static/clippy
+
 scripts/excmds_macros.py
 scripts/newtab.md.sh
 scripts/make_tutorial.sh
 scripts/make_docs.sh &
-nearleyc src/grammars/bracketexpr.ne > src/grammars/.bracketexpr.generated.ts
-native/install.sh local
 
-(webpack --display errors-only && scripts/git_version.sh)&
+nearleyc src/grammars/bracketexpr.ne \
+  > src/grammars/.bracketexpr.generated.ts
+
+if [ "$(isWindowsMingw)" == "True" ]; then
+  powershell native/win_install.ps1
```

Comment:
Please make this a local installation as I have mentioned on the PowerShell script.

Reply:
I think this is fixed as mentioned above.

The script also currently supports an additional [flag called
`-InstallDirBase`][0] to drop the installation files to any target
directory. The default is `%USERPROFILE%\.tridactyl`.

For example, the command below will install `native_main.py`,
`tridactyl.json` etc. in `C:\Users\<user-name>\Desktop\.tridactyl`
directory.

```
PS C:\Users\<user-name>\tridactyl> .\win_install.ps1 `
  -InstallDirBase C:\Users\<user-name>\Desktop
```

There's also [an `-Uninstall` flag][1], that removes these files and
the registry.

Example:

```
PS C:\Users\<user-name>\tridactyl> .\win_install.ps1 -Uninstall
```

Or,

```
PS C:\Users\<user-name>\tridactyl> .\win_install.ps1 `
  -Uninstall -InstallDirBase C:\Users\<user-name>\Desktop
```

[0]: https://github.com/gsbabil/tridactyl/blob/f301f6549c43aeace44e7e7e2daa1b2e70ce7af5/native/win_install.ps1#L111
[1]: https://github.com/gsbabil/tridactyl/blob/f301f6549c43aeace44e7e7e2daa1b2e70ce7af5/native/win_install.ps1#L42

Reply:
I think installing it locally from source should be done at this step i.e. `  PS C:\Users\<user-name>\tridactyl> .\win_install.ps1 -DebugDirBase .\native`.

Reply:
I get

```
$ npm run build

> tridactyl@0.1.0 build C:\Users\olie\projects\tridactyl
> sh scripts/build.sh

native/win_install.ps1 : File C:\Users\olie\projects\tridactyl\native\win_install.ps1 cannot be loaded because running
scripts is disabled on this system. For more information, see about_Execution_Policies at
https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ native/win_install.ps1
+ ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess
```

We should add a warning to the README or run `Set-ExecutionPolicy -ExecutionPolicy Unrestricted` in an elevated prompt.